File: 01-Languages/CSharp/18-Testing/MathHelpers.cs
// MathHelpers.cs - the module under test.

public class MathHelpers {
    public static int Add(int a, int b) => a + b;

    public static double Divide(double a, double b) {
        if (b == 0) throw new ArgumentException("Cannot divide by zero");
        return a / b;
    }
}

File: 01-Languages/Java/18-Testing/MathHelpers.java
// MathHelpers.java - the module under test.

public class MathHelpers {
    public static int add(int a, int b) {
        return a + b;
    }

    public static double divide(double a, double b) {
        if (b == 0) throw new ArithmeticException("Cannot divide by zero");
        return a / b;
    }
}
